src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/FileUploadReview.cshtml

```diff
@@ -77,6 +76,12 @@
                                 <label class="govuk-label govuk-radios__label" for="details-save">
                                     Save all but don't send to employer(s)
                                 </label>
+                                @if (AuthorizationService.IsAuthorized(ProviderFeature.RplExtended))
+                                {
+                                    <div id="details-save-hint" class="govuk-hint govuk-radios__hint">
+                                        You will need save first and then approve the cohorts. This is because the new RPL rules still need to be implemented here.
```

Comment:
Minor - but this doesn't read right so I suspect a word has been missed off.

Reply:
Will take this up with our tester later